### ISSUE TITLE
Question regarding ntpd plugin

### ISSUE THREAD

**Issue by dwdglb** [collectd/collectd#961]
Hi all, 

Apologies if my question is too trivial for this forum but couldn't find any information elsewhere - I'm collecting stats for my company's internal NTP servers using collectd version 5.1.0-3 but I find that metrics collected for time_dispersion are inconsistent with ntpq output for jitter. 

To illustrate it - whilst delay appear to be close to values returned by ntpq:
```
     remote           refid      st t when poll reach   delay   offset  jitter
==============================================================================
-85.119.80.232   129.250.35.250   3 u  112  128  377    1.339    0.822   0.112
```
values from csv output plugin: 

```delay > 1425566949.683,0.001328 = 1.328ms```

```time_dispersion > 1425566959.683,0.070145 =  70.145ms ?```

In this case - shouldn't time_dispersion be around 0.000112 to match values returned by ntpq ? 

If someone could please comment how time_dispersion is calculated by collectd I'd be most grateful as perhaps there's a gap in my understanding of how this metric is processed ?

Do I need a different (newer) version of collectd to be installed ? Any suggestions would be most welcome :)

Best Regards

**Comment by gflarity** [comment]
I've found that the values the plugin reports are completely different than when I got in and run:

ntpq -n -crv

I have a feeling it's a multiplication factor issue or something like that. I get this particular plugin isn't really maintained?

**Comment by mfournier** [comment]
This reminds me of #1300. Would you be able to check again with the tip of one of the release branches ? In case you use one of the supported distros, you can simply install the latest nightly build: https://github.com/collectd/collectd-ci/blob/master/README.md

**Comment by gflarity** [comment]
So I'm actually using a fork: https://github.com/Stackdriver/collectd

I don't have a lot of cycles unfortunately. If it's easy to test I'd be willing to help though.

**Comment by rubenk** [comment]
Yes, please test this with either 5.4 or 5.5.1, since this should have been solved by 53438c5

**Comment by octo** [comment]
Closing stale issue.
